orc8r/cloud/go/services/orc8r_worker/orc8r_worker/main.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"database/sql"
+
+	"magma/orc8r/cloud/go/services/orc8r_worker"
+
+	"github.com/golang/glog"
+
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/service"
+	"magma/orc8r/cloud/go/services/state"
+	state_config "magma/orc8r/cloud/go/services/state/config"
+	"magma/orc8r/cloud/go/services/state/indexer/reindex"
+	indexer_protos "magma/orc8r/cloud/go/services/state/protos"
+	"magma/orc8r/cloud/go/services/state/servicers"
+	"magma/orc8r/cloud/go/sqorc"
+	"magma/orc8r/cloud/go/storage"
+	"magma/orc8r/lib/go/service/config"
+)
+
+func main() {
+	srv, err := service.NewOrchestratorService(orc8r.ModuleName, orc8r_worker.ServiceName)
+	if err != nil {
+		glog.Fatalf("Error creating orc8r_worker service %v", err)
+	}
+
+	// TODO(reginawang3495): rename function name when jobQueue reindexer is removed
+	singletonReindex := srv.Config.MustGetBool(state_config.EnableSingletonReindex)
+	if singletonReindex {
+		startSingletonReindexer(srv)
+	}
```

Comment:
I would even move this config check into the "start reindexer" func. But can do in next PR